src/climate_learn/data/climate_dataset/era5_iterdataset.py

```diff
@@ -115,6 +116,16 @@ def __iter__(self):
                 + predict_ranges
             )
             outputs = y[output_ids]
+            
+            # print ('len orig', inputs.shape[0])
+            # print ('subsample', self.subsample)
+            
+            inputs = inputs[::self.subsample]
```

Comment:
Since the inputs are shuffled by time we are not really doing subsampling in time. Logically, we are just reducing the dataset size here.

Doing subsampling here would be a little tricky as you would need to know the preshuffled/tempral property of data points. Only then you can subsample it.